### PR TITLE
chore(i18n): Add missing Japanese rules translations

### PR DESCRIPTION
**Pull request by shunito**
Added translations of Japanese rules that are missing due to recent updates.

### CODE REVIEW

**Comment by straker** [comment]
Thanks for the pr. We'll need to look for someone who speaks Japanese that can review this, so might be a bit. But we'll make sure to get it merged.

File: locales/ja.json [hunk 1]
```diff
@@ -631,6 +668,7 @@
     },
     "focusable-not-tabbable": {
       "pass": "要素内にフォーカス不可能な要素は含まれていません",
+      "incomplete": "Check if the focusable elements immediately move the focus indicator",
```
Comment (shunito) [inline locales/ja.json:671]: Translation is not finished..

**Comment by shunito** [comment]
I found a section that has not been translated yet.
We will add them within a few days.

**Comment by shunito** [comment]
@straker  Thank you for your usual contributions.
I have added the missing translated text.
and, I have then asked the member who created the original Japanese translation to review it.

**Comment by ma10** [comment]
Maybe @kiyochika could review?

**Comment by straker** [comment]
Thanks for the translations. Approved for security.

**Pull request merged**